options/locale/locale_ru-RU.ini

```diff
@@ -173,6 +173,8 @@ app_url=Базовый URL-адрес Gitea
 app_url_helper=Этот параметр влияет на URL для клонирования по HTTP/HTTPS и на некоторые уведомления по электронной почте.
 log_root_path=Путь к журналу
 log_root_path_helper=Файлы журнала будут записываться в этот каталог.
+default_branch_name=Имя ветки по умолчанию
```

Comment:
will be overwritten ... please add them via crowdin after merge

Reply:
ok